expect.js/expect.js.d.ts

```diff
@@ -97,126 +149,123 @@ declare module Expect {
-         */
-        keys(keys: string[]): Assertion;
-        /**
-         * Assert exact keys or inclusion of keys by using the `.own` modifier.
+         * Assert that the function throws.
+         *
+         * @param regexp regexp to match error string against
          */
-        keys(...keys: string[]): Assertion;
+        throwError(regexp: RegExp): void;
 
         /**
-         * Assert a failure.
+         * Assert that the function throws.
+         *
+         * @param fn callback to match error string against
          */
-        fail(message?: string): Assertion;
+        throwException(regexp: RegExp): void;
     }
 
-    interface Root extends Assertion {
-        not: Not;
-        to: To;
+
+    interface ToBase {
+        be: Be;
+        have: Have;
+        include: Include;
         only: Only;
+    }
+
+    interface Only{
         have: Have;
-        be: Be;
     }
 
-    interface Be extends Assertion {
+    interface Include {
         /**
-         * Checks if the obj exactly equals another.
+         * Assert exact keys or inclusion of keys by using the `.own` modifier.
          */
-        (obj: any): Assertion;
+        key(keys: string[]): Assertion;
+        /**
+         * Assert exact keys or inclusion of keys by using the `.own` modifier.
+         */
+        key(...keys: string[]): Assertion;
 
-        an: An;
     }
-
-    interface An extends Assertion {
+    interface Have extends Include {
+        own: Assertion;
         /**
-         * Assert typeof.
+         * Assert property "length" exists and has value of n.
+         *
+         * @param n
          */
-        (type: string): Assertion;
+        length(n: number): Assertion;
 
         /**
-         * Assert instanceof.
+         * Assert property name exists, with optional val.
+         *
+         * @param name
+         * @param val
          */
-        (type: Function): Assertion;
-    }
-
-    interface Not extends Expect.NotBase {
-        to: Expect.ToBase;
-    }
-
-    interface NotBase extends Assertion {
-        be: Be;
-        have: Have;
-        include: Assertion;
-        only: Only;
-    }
-
-    interface To extends Expect.ToBase {
-        not: Expect.NotBase;
-    }
+        property(name: string, val?: any): Assertion;
 
-    interface ToBase extends Assertion {
-        be: Be;
-        have: Have;
-        include: Assertion;
-        only: Only;
-    }
+        /**
+         * Assert exact keys or inclusion of keys by using the `.own` modifier.
+         */
+        keys(keys: string[]): Assertion;
+        /**
+         * Assert exact keys or inclusion of keys by using the `.own` modifier.
+         */
+        keys(...keys: string[]): Assertion;
 
-    interface Only extends Assertion {
-        have: Have;
     }
+}
+declare function expect(typed?: any): e.Assert;
 
-    interface Have extends Assertion {
-        own: Assertion;
-    }
+declare module "expect" {
```

Comment:
@speedy32 Just this line breaks npm/commonjs loader.
You should report this issue to VS Code.